nodes/src/nodes/llm_gmi_cloud/IGlobal.py

```
@@ -0,0 +1,192 @@
+        """Initialize the GMI Cloud chat client."""
+        from depends import depends  # type: ignore
+
+        requirements = os.path.dirname(os.path.realpath(__file__)) + '/requirements.txt'
+        depends(requirements)
+
+        from .gmi_cloud import Chat
+
+        bag = self.IEndpoint.endpoint.bag
+        config = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+        if not config.get('apikey'):
+            raise ValueError('GMI Cloud API key is required.')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)